### DISCUSSION TITLE
Implementation of TEBD routine 2

### DISCUSSION THREAD

**Discussion by ngm29** [Part-II-Computational-Physics/.github#107] [Q&A] [answered]
For the TEBD project, the algorithm provided in the linked site is for infinite TEBD. Am I supposed to investigate finite TEBD too? If so I have found the following (https://github.com/tenpy/tenpy_toycodes/tree/main/tenpy_toycodes) implementation of the finite version, but I am unsure of whether I am allowed to use it or if I should implement my own.

**Comment by AustenLamacraft** [comment, marked as answer]
Yes, feel free to use it (properly attributed)
